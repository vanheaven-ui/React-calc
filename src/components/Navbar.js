import { Link } from 'react-router-dom';

const Navbar = () => (
  <nav className="navbar">
    <h2>Math Magicians</h2>
    <div className="links">
      <Link to="/">Home</Link>
      <Link to="/calc">Calculator</Link>
      <Link to="/quotes">Quotes</Link>
    </div>
  </nav>
);

export default Navbar;
