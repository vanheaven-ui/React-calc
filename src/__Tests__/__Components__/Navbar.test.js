import React from 'react';
import ReactDOM from 'react-dom';
import Navbar from '../../components/Navbar';

test('is rendered without crashing', () => {
  
});