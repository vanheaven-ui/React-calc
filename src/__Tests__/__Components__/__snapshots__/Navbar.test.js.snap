// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not change unexpectedly 1`] = `
<nav
  className="navbar"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <h2>
      Math Magicians
    </h2>
  </a>
  <div
    className="links"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/calc"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/quotes"
      onClick={[Function]}
    >
      Quotes
    </a>
  </div>
</nav>
`;
